fv/infrastructure/infra_k8s.go

```diff
@@ -236,7 +236,7 @@ func setupK8sDatastoreInfra() (*K8sDatastoreInfra, error) {
 	log.Info("Started controller manager.")
 
 	// Copy CRD registration manifest into the API server container, and apply it.
-	err := kds.k8sApiContainer.CopyFileIntoContainer("../vendor/github.com/projectcalico/libcalico-go/test/crds.yaml", "/crds.yaml")
+	err := kds.k8sApiContainer.CopyFileIntoContainer("infrastructure/crds.yaml", "/crds.yaml")
```

Comment:
Not sure if you want to go down this route, but you can do something like

```
cat $(go list -m -f "{{.Dir}}" github.com/projectcalico/libcalico-go)/test/crds.yaml
```

to get this content

Reply:
I spoke to Casey about the same issue on a different repo and we settled on curling these files from github, so I'm adding a rule for this one instead of adding the file.

Reply:
Ok, moved to `get list` instead